### ISSUE TITLE
Finetuning on BCIC-2A

### ISSUE THREAD

**Issue by HaneenSu** [BINE022/EEGPT#3]
Hello,

In your paper, you describe the approach of fine-tuning on BCIC-2A is slighlty differernt than that for other downstream datasets. 
<img width="1072" alt="image" src="https://github.com/user-attachments/assets/fa99db71-e386-4e7a-aae9-f811b58886ec">

However, that is not very clear within the text and in the github repo. I don't see a model called MSConvNet in the implementation and there is no finetune_EEGPT_BCIC2A.py file in the downstream folder. 

Some clarification is very much appreciated!

**Comment by whiteram** [comment]
We apologize for the oversight. In fact, in the final version submitted to NeurIPS 2024, we did not utilize the MSConvNet model. In a prior version, we had concatenated the pre-trained model (EEGPT) with a small-scale convolutional model (MSConvNet) specifically designed for motor imagery tasks, which indeed yielded better accuracy for downstream tasks. However, this approach was not conducive to a fair comparison and did not highlight the strengths of the EEGPT model. As a result, we have decided to discard this approach. We will update this in the final revision.

**The updated experimental configuration description, which the results in the main text are based on, is as follows:**

Experimental Configuration: Experiments using the LOSO method. In the BCIC2A dataset 4-category cross-subject experiment, the accuracy of the model in this paper is lower than that of the task-specific SOTA model. This may be related to the high individual variability of the BCIC2A data. 

The input data are 19-channel (FP1, FP2, F7, F3, FZ, F4, F8, T7, C3, CZ, C4, T8, P7, P3, PZ, P4, P8, O1 and O2 channels) EEG data with 4s, 256Hz sampling rate; to fully utilize the feature extraction capability of the model, spatial filters (convolution) were used to map the 19-channels of the input data into 19-channels (FP1, FP2, F7, F3, FZ, F4, F8, T7, C3, CZ, C4, T8, P7, P3, PZ, P4, P8, O1 and O2 channels) of the pre-trained encoder; the features are extracted using the pre-trained encoder and then mapped to category logits using the linear layer. 

~~The fine-tuning method of combining the task-specific model with the model in this paper was used for testing. The MSConvNet model, which has a multi-scale convolutional EEGNet-like structure, was used as the motor imagery task-specific model. The model branch first maps the input 22-channel EEG signal into 10-20 system standard channel data using an adaptive spatial filter (convolution); then a pretrained encoder is used to extract the generic representations; and after collocating the generic representations and the in-domain representations of Wenchao’s model branch, they are mapped into category logits using a linear layer.~~ 

Training is performed using the AdamW optimizer, OneCycle learning rate strategy [43] (starting learning rate 5e-4, maximum 1e-3, minimum 4.22e-7), 100 rounds of training, and batch size of 72.
